data_models/text.py

```diff
@@ -6,7 +6,7 @@
 class Text(_Content):
     """This class creates Text object"""
     TITLE_CONTENT_TYPES = [
-        "h1", "h2", "h3", "h4", "h5", "h6"
+        "h1", "h2", "h3", "h4", "h5", "h6", "title"
```

Comment:
Where is this change being used?

Reply:
1) we need to ensure the title is always extracted since we want to eliminate it from the list. The text could be classified as a title if its bold and within the word or it is of type `title` . This explicitly ensures this happens.
2) We take this title and make a stamp page with just that - so this will be used as a seed and will be automatically not added to the interesting sequence
hope this helps!

Reply:
Cool! Just amend the PR title to reflect this as well.